packages/ibm-products/src/components/PageHeader/PageHeader.stories.js

```diff
@@ -341,6 +341,7 @@ const title = {
   },
   'Long title with icon': {
     text: 'A very long page title which will almost certainly have to be truncated at some point',
+    shortText: 'A shortened title for nav',
```

Comment:
Placement here is a bit odd. Setting this as part of `title` does make sense, but I feel like I’d look for examples for this more explicitly. (I’d realistically probably look in breadcrumbs for `breadcrumb with shortened title` or something like that, but it might not make sense for how we’d actually set this.

I do think a separate option under title like `Long title with short name` might be a good alternative though.